index.html

```diff
@@ -1,12 +1,47 @@
 <!DOCTYPE html>
-  <html lang="en">
+  <html data-ng-app="todo">
   <head>
      <meta charset="UTF-8">
      <title>Angular ToDo App</title>
      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
      <script src="app.js"></script>
   </head>
-  <body>
+  <body data-ng-controller="todo_list as vm">
     <h1>To Do!</h1>
+    <table>
+      <thead>
+        <tr>
+          <th>
+            TASK LIST:
+          </th>
+          <th>
+            EDIT.
+          </th>
+          <th>
+            DONE?
+          </th>
+        </tr>
+      <tbody>
+        <tr data-ng-repeat="todo in vm.todoList">
+          <td>{{todo.task}}</td>
+
+          <td data-ng-click="editable=!editable">✎</td>
+          <td>
+            <form data-ng-show="editable">
+              <input type="text" data-ng-model="vm.taskContent">
+              <button data-ng-click="vm.update($index); editable=!editable">Update</button>
```

Comment:
Good job using a variety of different angular directives like `ng-model`, `ng-show`, `ng-click`, etc!